pkg/processor/build/builder.go

```diff
@@ -219,7 +227,12 @@ func (b *Builder) Build(options *platform.CreateFunctionBuildOptions) (*platform
 
 	// copy the configuration we enriched, restoring any fields that should not be leaked externally
 	enrichedConfiguration := b.options.FunctionConfig
-	enrichedConfiguration.Spec.Build.Path = b.originalFunctionConfig.Spec.Build.Path
+
```

Comment:
add a comment - function build path is irellevant whenever the code entry type is image  (especially when it is a local path)